templates/financial-services/thank-you.html

```diff
@@ -0,0 +1,32 @@
+{% extends "financial-services/_base_financial-services.html" %}
+
+{% block title %}Thank you | Financial services{% endblock %}
+{% block head_extra%}<meta name="robots" content="noindex" />{% endblock %}
+{% block meta_description %}Ubuntu enables sellside and buyside financial services institutions, retail banking operations and insurance and fintech firms to build secure, agile infrastructure with the flexibility to scale.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1qDjpWQx5T49danAFbH1KcbacV5UqK9c_LRKMF_cpOzM/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+
+  {% with thanks_context="contacting us about Ubuntu for financial services" %}{% include "shared/_thank_you.html" %}{% endwith %}
```

Comment:
This differs from the copy doc - "A member of our team will be in touch within one working day or you can [visit our store](https://buy.ubuntu.com/) to purchase UA Infrastructure."

Reply:
thanks, I missed that but fixed now